### ISSUE TITLE
Confirm/Fix markup validation errors 

### ISSUE THREAD

**Issue by klemay** [hypothesis/client#2059]
To meet WCAG 2.0 AA Criterion 4.1.1 (Parsing):

Our auditor noted: Markup validation errors present, such as missing and stray end tags, incorrectly structured elements and duplicate IDs.

NOTE:  Our developers have questions about this one, as we suspect the auditor may have been looking at markup errors for the content underneath the sidebar, rather than the client itself. I've connected Lyza with our auditor to sort this one out.

**Comment by lyzadanger** [comment]
If it's helpful, this is how I'm currently running validation on the client sidebar HTML. I use Chrome's Dev Tools, YMMV in other browsers.

1. Load page with embedded client in browser.
1. Open Dev Tools; go to `Elements` tab. I usually do this by right-clicking somewhere inside the sidebar and choosing "Inspect" from the contextual menu.
1. Find the `<html>` element that corresponds to the `iframe` that the sidebar is in—_not_ the top-level `html` element for the host page.
1. Right-click on that, select "Edit as HTML". That will make the `html` and all of its descendants selectable—the HTML document we care about
1. Select/copy all
1. Go to the HTML validator at https://validator.w3.org/#validate_by_input and paste the copied HTML here

**Comment by lyzadanger** [comment]
Now that #2110 has landed, we're down to only 2 errors from the W3C's HTML validator:

![image](https://user-images.githubusercontent.com/439947/81008980-cfd30700-8e21-11ea-9ca8-223a5edd1557.png)

After a conversation with @robertknight in Slack (https://hypothes-is.slack.com/archives/C1M8NH76X/p1588346972260100), I believe these errors are spurious: the `as` attribute is used correctly in our app per the HTML living standard, but the usage of it is not recognized by the W3C.

Thus, I believe we are done here!
